ts/features/fci/saga/index.ts

```diff
@@ -271,11 +275,29 @@ function* watchFciSignedDocumentsEndSaga(): SagaIterator {
   );
 }
 
+function* deletePath(path: string) {
+  yield RNFS.exists(path).then(exists =>
+    exists ? RNFS.unlink(path) : Promise.resolve()
+  );
+}
+
+/**
+ * Clears cached file for the fci document preview
+ * and reset the state to empty.
+ */
+function* clearAllFciFiles(action: ActionType<typeof fciClearAllFiles>) {
+  const path = action.payload.path;
+  if (path) {
+    yield* deletePath(path);
+  }
+}
+
```

Comment:
Looking at `fciClearAllFiles` definition path is non-nullable, is this check requied too?

```suggestion
function* clearAllFciFiles(action: ActionType<typeof fciClearAllFiles>) {
  yield* deletePath(action.payload.path);
}

```